lib/db/memory.js

```diff
@@ -135,12 +136,14 @@ MemoryStore.prototype = {
     delete this.clients[unbuf(id)];
     return P.resolve();
   },
-  generateCode: function generateCode(clientId, userId, email, scope) {
+  generateCode: function generateCode(clientId, userId, email, scope,
+                                      lastAuthAt) {
```

Comment:
Side note: I hate it when code gets this long. There is **no** good option of formatting, imo.